tests/test_pip_chill.py

```diff
@@ -10,12 +10,17 @@
 
 
 import os
+import platform
 import sys
 import unittest
 
-from pip_chill import pip_chill
+from pip_chill import pip_chill, cli
 from pip_chill.pip_chill import Distribution
 
+PIP_CHILL_CLI_FULL_PATH = cli.__file__
+PYTHON_EXE_PATH = sys.executable
+is_windows = any(platform.win32_ver())
```

Comment:
I'm reading TravisCI's Windows documentation and trying to figure out how it works. After this is merged, we should think about enabling Windows CI too.